### PR TITLE
Improve disasm_script.py and do some sample scripts

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

File: include/enums.h [hunk 1]
```diff
@@ -2486,6 +2496,83 @@ enum CameraFlags {
...
+    BTL_CAM_PRESET_A  = 0,
+    BTL_CAM_PRESET_B ,
+    BTL_CAM_PRESET_C ,
+    BTL_CAM_PRESET_D ,
+    BTL_CAM_PRESET_E ,
+    BTL_CAM_PRESET_F ,
+    BTL_CAM_PRESET_G ,
+    BTL_CAM_PRESET_H ,
+    BTL_CAM_PRESET_I ,
+    BTL_CAM_PRESET_9 ,
```
Comment (nanaian) [inline include/enums.h:2509]: Commas here

File: src/battle/common/actor/goomba.inc.c [hunk 2]
```diff
@@ -119,7 +119,7 @@ EvtSource N(idle_80219938) = {
     EVT_LOOP(LW(0))
         EVT_LABEL(0)
         EVT_CALL(GetStatusFlags, ACTOR_SELF, LW(1))
-        EVT_IF_FLAG(LW(1), 0x35D000)
+        EVT_IF_FLAG(LW(1), (STATUS_FLAG_SLEEP | STATUS_FLAG_FROZEN | STATUS_FLAG_FEAR | STATUS_FLAG_PARALYZE | STATUS_FLAG_DIZZY | STATUS_FLAG_STONE | STATUS_FLAG_STOP))
```
Comment (nanaian) [inline src/battle/common/actor/goomba.inc.c:122]: I'm pretty sure these parens aren't required anymore -- the double parens originally in flags were to tell the SCRIPT compiler things

**Review by nanaian — CHANGES_REQUESTED** [review]
Incredible

**Review by ethteck — APPROVED** [review]
looks great! One note to myself is that effects actually store their id in their main function, so I'll probably end up shifting all effects to use enum values instead of just numbers there.

**Review by nanaian — APPROVED** [review]
ty

**Pull request merged**